### PR TITLE
Get books published in the same year

### PR DESCRIPTION
**Pull request by schoure98**
I have added the code to recommend similar books based on the year of publication. Author, Publisher code is commented because creating issues while exporting python script. 
Exporting jupyter notebook as python script to include function getBooksYearly()
Please check the code and let me know if changes are required.

### CODE REVIEW

**Review by rutujapadgilwar — APPROVED** [review]
Everything looks good to me. Can you add the details in the comment why we are adding this recommendations.py?

**Comment by schoure98** [comment]
recommendations is a python file when recommendations.ipynb(jupyter file) is exported as python script. 
To get the books published in the same year I have created a getBooksYearly function. And need to use this function in app.py (python integration file).
Currently the use case of recommendations.py file is just to import the functions.

**Comment by schoure98** [comment]
Hi, I have added frontend code to display books when searched for year. Although styling is not added yet. 
@harshita-jaiswal can you make the styling changes as per required?

**Review by rutujapadgilwar — COMMENTED** [review]
I just tested your code on my local, it looks like there is no preview image for some books with no image URLs. Can you add a default image for the same?

File: templates/searchBooks.html [hunk 1]
```diff
@@ -1,24 +1,91 @@
...
-</html>
\ No newline at end of file
+		let user_input = document.querySelector("#user_input");
+		let divElement = document.querySelector(".title");
+		let title = document.querySelector(".recommendation-title");
+		title.textContent = `Books published in the same year`;
+		function submitForm() {
+			divElement.appendChild(title);
+		}
+	</script>
```
Comment (harshita-jaiswal) [inline templates/searchBooks.html]: As discussed, we can remove this script. No longer needed

Comment (schoure98) [reply templates/searchBooks.html]: Hi Harshita, removed the js script.

**Comment by schoure98** [comment]
@rutujapadgilwar I checked if we can add a sample image if url is not loading, it is happening because the given url is blank but its not empty. If there was no url it was possible to add through fillna()method. 
As per discussion with @harshita-jaiswal it might be possible to give default image through css possible, though not sure yet.

**Review by harshita-jaiswal — APPROVED** [review]

**Pull request merged**